project-base/config/packages/snc_redis.yaml

```diff
@@ -29,12 +29,12 @@ snc_redis:
             alias: 'global'
             dsn: 'redis://%redis_host%'
             options:
-                prefix: '%env(REDIS_PREFIX)%'
+                prefix: '%env(REDIS_PREFIX)%%build-version%'
```

Comment:
Hmm, I'm not sure if this solves the problem mentioned in the issue. 

Checks on CI (see `.ci/build_kubernetes.sh`) are run in the production environment anyway, so it will be still problematic.
Same we recommend in our documentation, https://docs.shopsys.com/en/9.0/cookbook/jenkins-configuration/ and 
indirectly in https://docs.shopsys.com/en/9.0/kubernetes/continuous-integration-using-kubernetes/

Internal projects run tests in the development environment, so for our teams it probably helps.
Maybe we should change the recommendations for running tests and how we run them ourselves.

Reply:
But checks do change the environment to test so it should be ok.